yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -475,3 +474,11 @@ func (p *ParserIssueDetector) getJsonbReturnTypeFunctions() []string {
 	jsonbFunctions = append(jsonbFunctions, catalogFunctionsReturningJsonb.ToSlice()...)
 	return jsonbFunctions
 }
+
+func (p *ParserIssueDetector) IsGinIndexPresentInSchema() bool {
+	return p.isGinIndexPresentInSchema
+}
+
+func (p *ParserIssueDetector) IsUnloggedTablesIssueFiltered() bool {
```

Comment:
curious - why this change from a field into a function?

Reply:
Nothing as such, just for it to be more sophisticated.